contrib/onionrouter.service

```diff
@@ -1,12 +1,24 @@
 [Unit]
 Description=onionrouter
 After=network.target
+Before=postfix.service
 
 [Service]
-ExecStart=/srv/onionrouter/onionrouter_run --config /srv/onionrouter/onionrouter/configs/onionrouter.ini --mappings /srv/onionrouter/onionrouter/configs/map.yml -p 23002
```

Comment:
Shouldn't the config and mappings path be passed somehow?

Reply:
even if the bin is `/usr/bin/onionrouter`, the rest of the options are still needed. right ?

Reply:
> Shouldn't the config and mappings path be passed somehow?

```
$ onionrouter --help
usage: onionrouter [-h] [--interactive] [--debug] [--client] [--config CONFIG] [--mappings MAPPINGS] [--host HOST] [--port PORT]

onionrouter daemon for postifx rerouting

optional arguments:
  -h, --help            show this help message and exit
  --interactive, -i     Simple test route mode without daemon
  --debug, -d           Debug mode. Run daemon and also print the queries & replies
  --client, -c          Client mode. Connect as a client to daemon for testing / debug
  --config CONFIG       Absolute path to config folder/file (default: /etc/onionrouter/)
  --mappings MAPPINGS   Absolute path to static mappings folder/file (default: /etc/onionrouter/mappings)
  --host HOST, -l HOST  Host for daemon to listen (default: 127.0.0.1)
  --port PORT, -p PORT  Port for daemon to listen (default: 23000)
```

The defaults for `config` and `mappings` are used if not specified, and those paths in `/etc` are very standard and are what users would expect, I think.



> even if the bin is `/usr/bin/onionrouter`, the rest of the options are still needed. right ?

see above. The defaults work great :+1: 

If a user wants to alter the command line, they can use `systemctl edit onionrouter.service` and provide different options or a different command entirely, like so:
```ini
[Service]
ExecStart=
ExecStart=/usr/bin/onionrouter --debug
```